conf/kubernetes-shuffle-service.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: spark-shuffle-service
+    version: 2.1.0
+  name: shuffle
+spec:
+  template:
+    metadata:
+      labels:
+        app: spark-shuffle-service
+    spec:
+      volumes:
+        - name: temp-volume
+          hostPath:
+            path: '/tmp'
```

Comment:
Makes sense. You're trying to lower the adoption barrier.

I would think `/spark-shuffle-files-0`, `/spark-shffule-files-1`, etc.